example-get-started-experiments/code/.github/workflows/dvc-studio.yml

```diff
@@ -56,10 +54,7 @@ jobs:
           DVCLIVE_LOGLEVEL: DEBUG
         run: |
           pip install -r requirements.txt
-          dvc pull
           cml ci --fetch-depth 0
```

Comment:
Have you tested without `--fetch-depth 0`? AFAIR we may not need it anymore, although I am not that confident about it.

Reply:
I have been actually encountering a weird error every time `cml ci` is used no matter the arguments. Need to look into it